projects/genie.lua

```diff
@@ -87,16 +87,19 @@ function defaultConfigurations()
 	configuration "Debug"
 		targetdir(BINARY_DIR .. "Debug")
 		defines { "DEBUG" }
+		buildoptions { "-std=c++11" }
```

Comment:
I'm not very sure about these options on MSVS in particular and about build configuration for Linux in general.